### PR TITLE
buld DB

### PR DESCRIPTION
**Pull request by anhdtp-1773**
=> Title: build DB
=> Redmine: https://edu-redmine.sun-asterisk.vn/issues/10768
=> Attachment File:

### CODE REVIEW

**Comment by anhdtp-1773** [comment]
ready

File: app/models/gender.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class Gender < ApplicationRecord
+	has_one :information_user
```
Comment (huuphuc93) [inline app/models/gender.rb]: indent

File: app/models/gender.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+	has_many :interested_in_gender
```
Comment (huuphuc93) [inline app/models/gender.rb]:
```suggestion
	has_many :interested_in_genders
```

File: app/models/user.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+	has_many :grade
```
Comment (huuphuc93) [inline app/models/user.rb]: - indent
- 
```suggestion
	has_many :grades
```

Comment (huuphuc93) [reply app/models/user.rb]: tương tự cho các has_many khác nhé

**Comment by anhdtp-1773** [comment]
ready

**Pull request merged**